boards/xtensa/intel_s1000_crb/intel_s1000_crb_defconfig

```diff
@@ -52,3 +52,8 @@ CONFIG_UART_NS16550=y
 CONFIG_AUDIO=y
 CONFIG_AUDIO_CODEC=y
 CONFIG_AUDIO_TLV320DAC=y
+
+CONFIG_SPI=y
```

Comment:
Do not enable those by default in the _defconfig, they should be set by the application.

https://docs.zephyrproject.org/latest/porting/board_porting.html#default-board-configuration

Reply:
done.